roles/ckan/tasks/azure-deploy.yml

```diff
@@ -0,0 +1,103 @@
+---
+
+- name: Get subscription_id for later
+  azure.azcollection.azure_rm_subscription_info:
+    all: true
+  register: subscription_output
+
+- name: Create CKAN namespace
+  command: >
+    az aks command invoke --resource-group {{ resource_group_name }} --name {{resource_prefix}}-aks \
```

Comment:
This should be done using `community.kubernetes.k8s` module, example [here](https://github.com/fjelltopp/fjelltopp-ansible/blob/795cad58b0ef0de3a456095540c33a20dea7def2/roles/ckan/tasks/deploy.yml#L2)

Reply:
Because the k8s cluster is a private instance, we don't have direct access to the k8s API, commands need to be proxied through the `az aks command invoke` command, which is far from ideal